chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/DeveloperManagerImpl.java

```diff
@@ -98,7 +117,7 @@
     }
 
     @Override
-    @Transactional(readOnly = true)
+    //@Transactional(readOnly = true)
```

Comment:
Since this is making a database call I would have expected the transactional annotation to be necessary.. can you explain why it's not?

Reply:
Not sure about this - might have commented it out when trying to debug other issues. I'll add it back in.